todo_app/trello.py

```diff
@@ -0,0 +1,80 @@
+import requests
+import os
+import json
+from todo_app.item import Item
+
+secret_key = os.getenv('SECRET_KEY')
+api_key = os.getenv('TRELLO_API_KEY')
+api_token = os.getenv('TRELLO_TOKEN')
+board_id= os.getenv('TRELLO_BOARD_ID')
+listid_todo = os.getenv('ID_LIST_TODO')
+listid_doing = os.getenv('ID_LIST_DOING')
+listid_done = os.getenv('ID_LIST_DONE')
+listid_newlist = os.getenv('ID_LIST_NEWLIST')
+# cards_cardOne =os.getenv('CARDS_TODO_CARD_ONE')
+# cards_cardTwo =os.getenv('CARDS_TODO_CARD_TWO')
+# cards_cardThree =os.getenv('CARDS_TODO_CARD_THREE')
+# cards_trelloDone =os.getenv('CARDS_TRELLODONE_CARD')
+# cards_trelloBoard =os.getenv('CARDS_DEVOPSTRELLOBOARD')
+
+
+class Trello():
+    
+    # Initializing the class, accepting Trello key and token as arguments.
+    def __init__(self):
+        self.url= "https://api.trello.com/1"
+        #self.url = "https://api.trello.com/1/boards/"
+
+    def getCardsfromList(self):
+        params= {'key': api_key, 'token': api_token}
+        response =  requests.get(f'https://api.trello.com/1/boards/{board_id}/cards', params= params)
+        print(response.text)
+        items=[]
+        for item in response.json():
+            if item['idList'] ==listid_todo:
+                status="ToDo"
+            elif item['idList'] == listid_done:
+                status ='Done'
```

Comment:
You've missed out a case:
Your app will currently break if there are any items with `item['idList'] == listid_doing`